<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulationView</class>
 <widget class="QWidget" name="simulationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
color:#0B0B0B;
font: 11pt &quot;OpenSans-Light&quot;;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="group_simulationParameters">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="title">
      <string>Simulation settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="la_population">
              <property name="text">
               <string>P</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_P">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0);
corder-color: rgba(0,0,0,0);</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="la_nbOfInfected">
              <property name="text">
               <string>I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_I">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0);
corder-color: rgba(0,0,0,0);</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="la_simulationTime">
              <property name="text">
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_D">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0);
corder-color: rgba(0,0,0,0);</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_send_PID">
           <property name="text">
            <string>Send PID parameters</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_graphType">
        <property name="title">
         <string>Plot graphs</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="graphStyleWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="cb_voltage">
              <property name="text">
               <string>voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_current">
              <property name="text">
               <string>current</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="cb_temperature">
              <property name="text">
               <string>temperature</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Command:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fittingSimulationWidget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QWidget" name="widget_5" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_10" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="la_cmdWaitTime">
                 <property name="text">
                  <string>Wait Time (ms)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_cmdWaitTime">
                 <property name="text">
                  <string>500</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Min:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_min">
                 <property name="text">
                  <string>25</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Max:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_max">
                 <property name="text">
                  <string>26</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>nbOfPoints</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="le_nbOfPoints">
                 <property name="text">
                  <string>100</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QRadioButton" name="rb_triangle">
                 <property name="text">
                  <string>Triangle Wave</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>Sinus Wave</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_3">
                 <property name="text">
                  <string>Square Wave</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_loop">
              <property name="text">
               <string>Loop Command</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="GraphicsLayoutWidget" name="pyqtgraphWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_11" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QPushButton" name="pb_resetConnection">
        <property name="text">
         <string>Reset Connection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pb_clearGraph">
        <property name="text">
         <string>Clear Graph</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_read">
        <property name="text">
         <string>Read Values</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_launchCommand">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Send Command</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pb_stopCommand">
        <property name="text">
         <string>Stop Command</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pb_kill">
        <property name="text">
         <string>Kill Routine Thread</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
